src/ontology/slurp/doid.tsv

```diff
@@ -29,997 +29,986 @@ MONDO:0850026	hepatobiliary system cancer	DOID:0080355	MONDO:equivalentTo	hepato
 MONDO:0850027	malignant adenoma	DOID:0080364	MONDO:equivalentTo	malignant adenoma	A cell type cancer that is composed_of epithelial tissue in which tumor cells form glands or glandlike structures, representing an early form of colorectal cancer.	MONDO:0004992
 MONDO:0850028	testicular sex cord-stromal benign neoplasm	DOID:0080371	MONDO:equivalentTo	testicular sex cord-stromal benign neoplasm	A sex cord-stromal benign neoplasm that arises from the testis.	MONDO:0024988
 MONDO:0850029	epithelioid inflammatory myofibroblastic sarcoma	DOID:0080372	MONDO:equivalentTo	epithelioid inflammatory myofibroblastic sarcoma		MONDO:0015798
-MONDO:0850030	epididymis disease	DOID:0080373	MONDO:equivalentTo	epididymis disease	A male reproductive system disease that is located_in the epididymis.	MONDO:0003150
-MONDO:0850031	gastroesophageal cancer	DOID:0080374	MONDO:equivalentTo	gastroesophageal cancer		MONDO:0002516
```

Comment:
@matentzn Want your advice on big diffs in slurp TSVs.

### Sorting not a prob
These are not happening because unsorted. They've been sorted like this:
```py
    result = result.sort_values(['mondo_id', 'mondo_label', 'xref', 'xref_source', 'original_label', 'definition', 'parents'])
```

### Non-persistent candidate Mondo IDs
These diffs are big is because of something else. See: 
- #216